support-frontend/assets/helpers/contributions.ts

```diff
@@ -282,9 +282,33 @@ function getConfigAbTestMin(
 ): number {
 	if (contribType === 'ANNUAL') {
 		if (variantData.variantA) {
-			return 30;
+			if (
+				countryGroupId === 'AUDCountries' ||
+				countryGroupId === 'NZDCountries'
+			) {
+				return 40;
+			} else if (countryGroupId === 'Canada') {
+				return 35;
+			}
+			if (
+				countryGroupId === 'EURCountries' ||
+				countryGroupId === 'International'
+			) {
+				return 30;
+			} else {
+				return 25;
+			}
```

Comment:
Would it be clearer to write this as a switch? I think the following should work:

```suggestion
			switch (countryGroupId) {
				case 'AUDCountries':
				case 'NZDCountries':
					return 40;
				case 'Canada':
					return 35;
				case 'EURCountries':
				case 'International':
					return 30;
				case 'GBPCountries':
				case 'UnitedStates':
					return 25;
			}
```

That way there’s less repetition of the variable name, and all cases are named (don’t have to work out that the default case is UK/US). I think the exhaustiveness checking should still be able to tell that we don’t need a default case.

Reply:
Yes, sorry had not finished, thought would send out conversions request in a bit on the chat and this wouldnt come up on peoples radar till then.  Will refine now, got to rebase with checkout minimum fix as well first.

Reply:
done fix

Reply:
Ah sorry! I didn’t realise it wasn’t ready for review yet